attributes/default.rb

```diff
@@ -4,7 +4,7 @@
 #
 # Copyright 2014-2016, Bloomberg Finance L.P.
 #
-
+default['go']['version'] = '1.7'
```

Comment:
I'm trying a few things, retesting - removing that, running tests, had git-centos-* failures similar:
```
    command/agent/command.go:39:2: cannot find package "github.com/hashicorp/go-multierror" in any of:
           	/usr/local/go/src/github.com/hashicorp/go-multierror (from $GOROOT)
           	/opt/go/src/github.com/hashicorp/go-multierror (from $GOPATH)
```
Testing a few other things right now.